tests/components/switch/test_init.py

```diff
@@ -54,7 +59,8 @@ def test_methods(self):
         self.hass.block_till_done()
 
         assert not switch.is_on(self.hass)
-        assert STATE_OFF == self.hass.states.get(switch.ENTITY_ID_ALL_SWITCHES).state
+        assert STATE_OFF == self.hass.states.get(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)